src/pages/components/progress-bar/usage.mdx

```diff
@@ -0,0 +1,504 @@
+- To convey that data is being requested, transferred or processed
+
+#### When not to use
+
+- When the progress bar will be replaced by a minor user interface element once
+  loaded, use [skeleton states](/patterns/loading) instead
+- When manual user actions are required to progress, use a
+  [progress indicator](/components/progress-indicator/usage) instead
+- When the progress is determined by user actions such as tutorials completed,
+  or storage space, rather than system actions
+- If the process takes less than 10 seconds to load, use
+  [loading spinners](/components/loading/usage) instead
+
+#### Loading, inline loading, or progress bar?
+
+- If an activity displays a loading spinners on one screen, it should not show
+  the progress bar elsewhere in the design.
+- Both progress bars and circles can show either determinate or indeterminate
+  progress. The given area should help determine if a progress bar or circle is
+  best for that space. Progress bars are preferred in vertically narrow areas
+  (tables, cards, etc.) while progress circles are ideal for full page loading
+  or in very small areas.
+
+## Live demo
+
+## Variants
+
+A progress bar can either be determinate or indeterminate. The determinate
+variant is visualized as a track on which a bar fills up from left to right
+representing the progress. The indeterminate variant is visualized as a track on
+which a bar is moving along in constant speed, repeating over time.
+
+When it is neccesary to show the percentage and the descriptive text, the
+developers will have to calculate the percentage on their side with the value:
+`${(value/max*100).toFixed(2)}% (${value} of ${max} items)`
+
+| Variant                        | Purpose                                                                                                                                                                                                                                                    |
+| ------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
+| [Determinate](Determinate)     | Use determinate progress bars for a specified amount of time or clear information about the progression — they convey exact and qualitative information about the progression.                                                                             |     |
+| [Indeterminate](Indeterminate) | Use indeterminate progress bars for an unspecified completion of time or unclear information about the progression — they convey that users’ request, action, or data is being processed at the moment without indicating how long the activity will take. |
+
+<Row>
+<Column colLg={4}>
+
+![Determinate progress bars fill from 0 to 100%.](images/progress-bar-usage-2.gif)
```

Comment:
Gifs need to use the [gif component](https://gatsby-theme-carbon.vercel.app/components/GifPlayer) so that it has a pause button on them.  

```
<Column` colLg='4'>
<GifPlayer>

![IBM Cloud Pictograms](/images/IBM_Cloud_Pictograms.gif) // must be gif

![IBM Cloud Pictograms](/images/IBM_Cloud_Pictograms.jpg) // must be static
image

</GifPlayer>
```

Reply:
@thyhmdo @laurenmrice this is why the gif is disappearing on pause - a static image needs to be given.